### ISSUE TITLE
DICOM: Creation of https://w3id.org/dcm namespace

### ISSUE THREAD

**Issue by ebremer** [w3c/hcls-fhir-rdf#151]
I suggest we create a https://w3id.org/dcm namespace until an official DICOM namespace could be assigned.  I don't mind using https://halcyon.is/dicom/ns/, but I thought https://w3id.org/dcm (or https://w3id.org/dicom) would be less confusing for anyone scrutinizing the work.  Any thoughts on this?

**Comment by dbooth-boston** [comment]
AGREED: We'll continue working on DICOM RDF, and bring it to the DICOM org when we feel like it is mature enough.  Hopefully then an official DICOM RDF group could be created, and they could agree to allocate a DICOM IRI stem.

**Comment by dbooth-boston** [comment]
Discussed today: https://www.w3.org/2025/01/09-hcls-minutes.html
